sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -371,7 +371,7 @@ export class InstitutionController extends BaseController {
   @Get("/my-locations")
   async getMyInstitutionLocations(
     @UserToken() userToken: IInstitutionUserToken,
-  ): Promise<InstitutionLocationsDetailsDto[]> {
```

Comment:
There were 2 interfaced defined with the same name InstitutionLocationsDetailsDto in institution-location.dto.

Hence renamed them to be easily notifiable.